simplified-ekirjasto-testing/src/main/java/fi/kansalliskirjasto/ekirjasto/testing/TestLoginFragment.kt

```diff
@@ -0,0 +1,148 @@
+package fi.kansalliskirjasto.ekirjasto.testing
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import android.view.ViewGroup
+import android.view.inputmethod.EditorInfo
+import android.view.inputmethod.InputMethodManager
+import android.widget.EditText
+import android.widget.ProgressBar
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import fi.ekirjasto.testing.BuildConfig
+import fi.ekirjasto.testing.R
+import org.nypl.simplified.android.ktx.supportActionBar
+import org.slf4j.LoggerFactory
+
+
+/**
+ * A fragment for browsing and reading digital magazines.
```

Comment:
Maybe change this to reflect actual purpose

Reply:
Done.